docs/api/role-management/put.asciidoc

```diff
@@ -74,60 +74,32 @@ Grant access to various features in all spaces:
 --------------------------------------------------
 $ curl -X PUT api/security/role/my_kibana_role
 {
-  "metadata" : {
-    "version" : 1
+  "metadata": {
+    "version": 1
   },
   "elasticsearch": {
-    "cluster" : [ ],
-    "indices" : [ ]
+    "cluster": [ ],
+    "indices": [ ]
   },
   "kibana": [
     {
-      "base": [],
+      "base": [ ],
       "feature": {
-       "discover": [
-          "all"
-        ],
-        "visualize": [
-          "all"
-        ],
-        "dashboard": [
-          "all"
-        ],
-        "dev_tools": [
-          "read"
-        ],
-        "advancedSettings": [
-          "read"
-        ],
-        "indexPatterns": [
-          "read"
-        ],
-        "graph": [
-          "all"
-        ],
-        "apm": [
-          "read"
-        ],
-        "maps": [
-          "read"
-        ],
-        "canvas": [
-          "read"
-        ],
-        "infrastructure": [
-          "all"
-        ],
-        "logs": [
-          "all"
-        ],
-        "uptime": [
-          "all"
-        ]
+       "discover": [ "all" ],
+        "visualize": [ "all" ],
+        "dashboard": [ "all" ],
+        "dev_tools": [ "read" ],
+        "advancedSettings": [ "read" ],
+        "indexPatterns": [ "read" ],
+        "graph": [ "all" ],
+        "apm": [ "read" ],
+        "maps": [ "read" ],
+        "canvas": [ "read" ],
+        "infrastructure": [ "all" ],
+        "logs": [ "all" ],
+        "uptime": [ "all"  ]
```

Comment:
Sorry for the noise here but the formatting was all over the place so picked one of the conventions and aligned the document. 

Also condensed the curl examples a bit since they were unnecessarily stretched out.